placement/algo/helper.go

```diff
@@ -227,7 +229,7 @@ func (ph placementHelper) getShardLen() int {
 	return len(ph.uniqueShards)
 }
 
-func (ph placementHelper) canAssignRack(shard uint32, from placement.HostShards, toRack string) bool {
+func (ph placementHelper) CanMoveShardToRack(shard uint32, from placement.HostShards, toRack string) bool {
```

Comment:
Yeah I meant the new function name would be more consistent with its purpose because it seemed a bit weird if `CanMoveShardToRack` returns true and we assign the shard to the rack anyway.